cypress/integration/account/change-password.spec.js

```diff
@@ -55,8 +55,8 @@ describe('OAuth change password', () => {
       .should('contain.text', 'Confirm password is required')
       .and('be.visible')
 
-    cy.location('pathname').should('eq', `${Cypress.config('oauthRootPath')}/update-password`)
-  })
+    cy.location('pathname').should('eq', `${Cypress.config('accountAppRootPath')}/update-password`)
+  }).skip()
```

Comment:
why?

Reply:
These are the old tests for the authenticated password change, which is currently still missing. I'll unskip them together with the profile settings PR.